src/Model/FieldType/EmailField.ts

```diff
@@ -1,9 +1,7 @@
 import { IFieldType } from "./IFieldType";
-import { isString } from "validate.js";
+import { StringField } from "./StringField";
 
 export const EmailField: IFieldType<string> = {
-  serialize: true,
+  ...StringField,
   defaultValidationRules: { email: true },
-  defaultValue: "",
-  isValidType: (value) => value == null || isString(value),
```

Comment:
Where is this getting its `normalize` function from?

Reply:
It was updated to use the ES6 `spread` operator, so the properties from the `StringField` are being copied onto the object, as if `EmailField` were extending `StringField`.

Reply:
Ah, makes sense.  No change needed, then.